src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -1172,9 +1172,11 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
             responseRows.add(responseRow);
         }
         
-        responseRows.addAll(getRemainingMissingResponseRows(question, possibleGiversWithoutResponses, 
-                                                            possibleReceiversWithoutResponsesForGiver, 
-                                                            prevGiver, viewType));
+        if (!responses.isEmpty()) {
+            responseRows.addAll(getRemainingMissingResponseRows(question, possibleGiversWithoutResponses, 
+                                                                possibleReceiversWithoutResponsesForGiver, 
+                                                                prevGiver, viewType));
+        }
```

Comment:
Hmm, this method is specific to every question. So it will help if there's only responses to a single question.